test/IDE/module-aliasing-repl-code-complete.swift

```diff
@@ -0,0 +1,46 @@
+/// Test REPL code completion with module aliasing
+/// When -module-alias <alias_name>=<real_name> is applied, code completion should show
+/// the <alias_name> as that's the name which should appear in source files including import statements,
+/// decls, expressions, etc. while getting visible decls come from the module of <real_name>, which
+/// is the name of the binary.
+/// Below, XLogging is the alias and mapped to the real name AppleLogging. Note that the binary name
+/// AppleLogging should not appear in the code completion results.
+///
+// RUN: %empty-directory(%t)
+// RUN: %{python} %utils/split_file.py -o %t %s
+
+// RUN: %target-swift-frontend %t/FileLogging.swift -module-name AppleLogging -module-alias XLogging=AppleLogging -emit-module -o %t/AppleLogging.swiftmodule
+
+// RUN: %target-swift-ide-test -repl-code-completion -source-filename %t/FileLib1.swift -module-alias XLogging=AppleLogging -I %t > %t/result1.txt
+// RUN: %FileCheck %s < %t/result1.txt
+
+// RUN: %target-swift-ide-test -repl-code-completion -source-filename %t/FileLib2.swift -module-alias XLogging=AppleLogging -I %t > %t/result2.txt
+// RUN: %FileCheck %s < %t/result2.txt
+
+// CHECK: Begin completions
+// CHECK-NOT: AppleLogging
+// CHECK-DAG: XLogging
```

Comment:
It doesn't really matter, but since you're now only checking for one thing (excluding the CHECK-NOTs), the -DAG doesn't really do anything.  The goal with DAG is for multiple checks with arbitrary order, e.g.

```
// CHECK: MustComeFirst
// CHECK-DAG: AnywhereBetweenFirstAndLast1
// CHECK-DAG: AnywhereBetweenFirstAndLast2
// CHECK: MustComeLast
```